frontend/src/components/Psychologist/Reimbursement/Billing.jsx

```diff
@@ -61,15 +72,78 @@ const Billing = () => {
       </Callout>
       <div className="fr-my-2w">
         <h3>Générer ma facture</h3>
+        {!canGenerateBill && (
+        <Notification type="info">
+          Veuillez attendre la signature de votre convention avant d&lsquo;envoyer votre facture.
+          Renseignez le statut de votre convention dans la page
+          {' '}
+          <HashLink to="/psychologue/mon-profil">Mes informations</HashLink>
+        </Notification>
+        )}
         <div className={styles.monthPickerContainer}>
           Générer ma facture pour le mois de :
           <div className={styles.monthPicker}>
             <MonthPicker month={month} setMonth={setMonth} />
           </div>
         </div>
-
         {filteredDate.length > 0 ? (
           <>
+            <div className="fr-mb-2w">
+              {fillInfo && (
+                <BillingInfo
+                  billingInfo={billingInfo}
+                  setBillingInfo={setBillingInfo}
+                />
+              )}
+            </div>
+            <ButtonGroup className="fr-mb-2w" isInlineFrom="xs">
+              {fillInfo ? (
+                <Button
+                  secondary
+                  icon="fr-fi-close-line"
+                  onClick={() => {
+                    setBillingInfo(billingInfoService.get());
+                    setFillInfo(false);
+                  }}
+                >
+                  Annuler
+                </Button>
+              ) : (
+                <Button
+                  secondary
+                  icon="fr-fi-edit-line"
+                  onClick={() => setFillInfo(true)}
+                >
+                  Renseigner mes informations
+                </Button>
+              )}
+              {canGenerateBill ? (
+                <a
+                  className="fr-btn"
+                  href={`/psychologue/bill/${month.month}/${month.year}`}
+                  target="_blank"
+                  rel="noreferrer"
+                  onClick={() => {
+                    setFillInfo(false);
+                    billingInfoService.save(billingInfo);
+                  }}
+                >
+                  <span className={classNames(styles.downloadIcon, 'fr-fi-file-download-line')} aria-hidden="true" />
+                  Télécharger/Imprimer
+                </a>
+              ) : (
+                <Button
+                  disabled
+                  icon="fr-fi-file-download-line"
+                >
+                  Télécharger/Imprimer
+                </Button>
```

Comment:
Too bad we can't declare same Button for both cases and simply `disabled: {canGenerateBill}`